pyproject.toml

```diff
@@ -69,6 +69,7 @@ Flask-Cors = "^3.0.10"
 pdoc = "^12.2.0"
 dateparser = "^1.1.4"
 schematic-db = {version = "^0.0.6", extras = ["synapse"]}
+pyopenssl = "^23.0.0"
```

Comment:
This will be handled by the other PR right?

Reply:
Yes